src/internal/modules/charge-information/controllers/charge-information-workflow.js

```diff
@@ -4,26 +4,21 @@ const { hasScope } = require('internal/lib/permissions');
 const { deleteChargeInfo } = require('../forms');
 const services = require('../../../lib/connectors/services');
 const { sortBy } = require('lodash');
-const getChargeVersionWorkflowsByStatus = (workflows, status) =>
-  workflows.filter(workflow => workflow.status === status);
-
-const getChargeVersionWorkflowsForTabs = workflows => ({
-  toSetUp: getChargeVersionWorkflowsByStatus(workflows, 'to_setup'),
-  review: getChargeVersionWorkflowsByStatus(workflows, 'review'),
-  changeRequest: getChargeVersionWorkflowsByStatus(workflows, 'changes_requested')
-});
 
 const getChargeInformationWorkflow = async (request, h) => {
-  const { toSetUp, review, changeRequest } = getChargeVersionWorkflowsForTabs(request.pre.chargeInformationWorkflows);
+  const toSetUp = request.pre.chargeInformationWorkflows;
+  const review = request.pre.chargeInformationWorkflowsReview;
+  const changeRequest = request.pre.chargeInformationWorkflowsChangeRequest;
+
   const view = {
     back: '/manage',
     ...request.view,
     pageTitle: 'Charge information workflow',
-    licences: { changeRequest, toSetUp, review: sortBy(review, ['chargeVersion.dateRange.startDate']) },
+    licences: { changeRequest, toSetUp, review: { data: [...sortBy(review.data, ['chargeVersion.dateRange.startDate'])], pagination: review.pagination } },
```

Comment:
Not important: Please consider breaking up the stuff on line 17 into multiple lines to make it more readable.